support-metrics-common/src/test/java/io/confluent/support/metrics/BaseSupportConfigTest.java

```diff
@@ -38,6 +38,21 @@ public void testValidCustomer() {
     }
   }
 
+  @Test
+  public void testValidNewCustomer() {
+    String[]
+        validNewCustomerIds =
+        ObjectArrays.concat(CustomerIdExamples.validCaseSensitiveNewCustomerIds,
+                            CustomerIdExamples.validCaseInsensitiveNewCustomerIds,
+                            String.class
+        );
+    for (String validId : validNewCustomerIds) {
```

Comment:
negative test cases? matching `.*` would work for these as well, but wouldn't be correct

Reply:
Sorry, I did not understand your comment. We are testing that invalid customer IDs are not accepted below. Or what other negative test can we do?

Reply:
you're not missing anything, filed comment as i read through and existing test + new invalid IDs just hadn't been spotted yet. additional invalid examples, combined with existing test, seem more than sufficient. sorry for the confusion!